module_shim.js

```diff
@@ -0,0 +1,3 @@
+"use strict";
+
+var module = {};
```

Comment:
Borrowed this from webcompat-addon to make it  so `module` do not error out in the browser (added `module.exports` so I could test some functions)